__tests__/component/form/EditTimesheetForm.test.tsx

```diff
@@ -0,0 +1,208 @@
+import React from "react"
+import { render, screen, fireEvent, waitFor } from "@testing-library/react"
+import "@testing-library/jest-dom"
+import sinon, { spy } from "sinon"
+import { Timesheet } from "@/lib/types/apiTypes"
+import { ApiUpdateResponse } from "@/lib/types/hooks"
+import { EditTimesheetForm } from "@/components/form/TimesheetForm"
+import { NEXT_PUBLIC_API_URL } from "@/lib/conf"
+import {
+    testEmployee,
+    testProject,
+    testTimesheet,
+    testTimesheetAllFields,
+    testTimesheetRequiredFields,
+} from "../../fixtures"
+
+// eslint-disable-next-line id-match, id-length
+import _ from "lodash"
+
+const customerEndpointURL = `${NEXT_PUBLIC_API_URL}/timesheet`
+const bodySpy = sinon.spy((body) => body)
+const pathSpy = sinon.spy((path) => path)
+
+jest.mock("@/lib/utils/fetch", () => ({
+    // eslint-disable-next-line require-await
+    put: async (
+        path: string,
+        body: object
+    ): Promise<ApiUpdateResponse<Timesheet>> => pathSpy(path) && bodySpy(body),
+}))
+
+afterEach(() => {
+    bodySpy.resetHistory()
+    pathSpy.resetHistory()
+})
+
+const fillAndSubmitForm = async (timesheet: Timesheet) => {
+    const nameInput = screen.getByTestId("form-field-name")
+    if (timesheet.name) {
+        fireEvent.change(nameInput, { target: { value: timesheet.name } })
+    } else {
+        fireEvent.change(nameInput, { target: { value: "" } })
+    }
+
+    const descriptionInput = screen.getByTestId("form-field-description")
+    if (timesheet.description) {
+        fireEvent.change(descriptionInput, {
+            target: { value: timesheet.description },
+        })
+    }
+
+    const allocationInput = screen.getByTestId("form-field-allocation")
+    if (timesheet.allocation) {
+        fireEvent.change(allocationInput, {
+            target: { value: timesheet.allocation },
+        })
+    }
```

Comment:
I think this should have a similar else statement to this

https://github.com/three-consulting/epoc-frontend/blob/07eba6d33d50cd476478145b33ec80982420e2c0/__tests__/component/form/EditTimesheetForm.test.tsx#L41-L43

Reply:
Actually it's probably better to remove these if's and replace them with the one-liners like
```typescript
fireEvent.change(nameInput, { target: { value: timesheet.name || "" } })
```